### ISSUE TITLE
User story - Tech ordering

### ISSUE THREAD

**Issue by stephendawsondev** [stephendawsondev/StackPortfolio#41]
### If applicable, add a related Epic.

#45 

### User Story

As a **tech user**, I can **order the tech on my projects** so that I can **give importance to that tech**.

### Acceptance Criteria

- User should be able to drag and drop tech on projects to order it by how important it is

### Please ensure you've completed these tasks before marking the issue as done.

- [x] Reviewed the user story for clarity and completeness.
- [x] Identified and linked related issues or epics.
- [x] Created or updated the necessary documentation.
- [x] Written unit and/or integration tests for functionality.
- [x] All code changes have been manually tested to ensure they meet the acceptance criteria.
- [x] Updated the issue with any relevant notes or feedback (to date).
- [x] Merged into the main branch and deployed to production (if applicable).

### Please add any further tasks that need to be completed before the issue can be marked as done.

- [x] Add a row to the Intermediary table `ProjectTech` that holds the order of the tech
- [x] Update UI so that the order can be changed
- [x] Ensure that tech on the project table is added, updated and deleted as expected - find out if there is a way to add tech ordering that would allow Django to keep doing that automatically.

**Comment by stephendawsondev** [comment]
This is actually something that sounded easy, but may actually be difficult. Django automatically sets up the intermediary table, which is great, but if I want to add tech ordering, I'd need to take more control of that table. The issue is that would remove the automatic creation/updating/deleting (unless I can find a way around that), so I'm marking this as an 8.

**Comment by stephendawsondev** [comment]
After investigating this further, it looks like this will be too troublesome to implement in this sprint - I'm marking it as `won't have` and moving it to the future features column. Here's the discussion that spurred the decision: https://forum.djangoproject.com/t/changing-a-manytomanyfield-to-use-a-through-model/10581

**Comment by stephendawsondev** [comment]
I have done this for projects, but I don't think it's necessary to do for job posts, and for profiles I want to do it based on how many times that tech appears across the different projects.
